baseapp/baseapp.go

```diff
@@ -426,7 +426,6 @@ func (app *BaseApp) StoreConsensusParams(ctx sdk.Context, cp *abci.ConsensusPara
 	if app.paramStore == nil {
 		panic("cannot store consensus params with no params store set")
 	}
-
```

Comment:
This would raise a `wsl` warning BTW.

Reply:
what is a `wsl`?

Reply:
See https://github.com/cosmos/cosmos-sdk/pull/7674#discussion_r512814802. Tiny suggestion, feel free to ignore it.